sockeye/data_io.py

```diff
@@ -262,7 +262,7 @@ def smart_open(filename: str, mode: str = "rt", ftype: str = "auto", errors:str
         return open(filename, mode=mode, encoding='utf-8', errors=errors)
 
 
-def read_content(path: str, limit: bool = None) -> Iterator[List[str]]:
+def read_content(path: str, limit: int = None) -> Iterator[List[str]]:
```

Comment:
Good catch, but could you change to "Optional[int]" since its set to None by default?

Reply:
Done